### DISCUSSION TITLE
If I only have the 10x matrix files for scATAC and don't have the fragments file, can I use Signac to analyze the scATAC data? How should I do it?

### DISCUSSION THREAD

**Discussion by szbssg** [stuart-lab/signac#1757] [Q&A] [answered]
When I wanted to analyze the scATAC data from GSE169246, I found that the authors only provided the mtx files and did not provide the fragments file. I encountered many difficulties during the analysis. I would like to ask if it is possible to generate a fragments file from the 10xATAC mtx files, or how should I proceed with the analysis?

**Comment by timoast** [comment, marked as answer]
You cannot generate a fragment file from the matrix. You can read about the fragment file format here: https://www.10xgenomics.com/support/software/cell-ranger-arc/latest/analysis/fragments-file

**Comment by Jyoung-0108** [comment]
哈喽，你找到解决办法了吗？我也想分析这个数据集，发现没有fragments文件

**Comment by szbssg** [comment]
没有那个文件就没辙


---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年11月18日 19:43 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [stuart-lab/signac] If I only have the 10x matrix files for scATAC and don't have the fragments file, can I use Signac to analyze the scATAC data? How should I do it? (Discussion #1757) |

哈喽，你找到解决办法了吗？我也想分析这个数据集，发现没有fragments文件
